Software/Human-Zombie-Device/DeviceState.h

```diff
@@ -6,6 +6,21 @@
 #include <cstring>
 #include <stdbool.h>
 
+//TODO: If these are to be configurable items they need to be in a struct not #defines
+#define MIN_DETECTION_RSSI_STENGTH -70
+#define NUM_OF_INFECT_HITS 3
+#define HEALTH_REDUCTION 1
+#define HEALTH_MAX 2
+#define HEALTH_MIN 0
+
+
+struct HealthBar {
+    uint health = HEALTH_MAX; // This is something that should probably live in EEPROM too... I think we need an EEPROM layout 
+};
+
+extern uint numberOfInfectionTicks;
```

Comment:
*Ideally* - > We should be moving to using Getters, Setters instead of sharing these via externals. 
Really helps move to build out the interfaces a bit better.

Reply:
yup, but thats a future issue not a now issue. I dont think it shouldnt be merged into main because of that